iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/api/OutputDeclaration.java

```diff
@@ -19,5 +19,22 @@
  */
 package org.sonar.iac.arm.tree.api;
 
-public interface OutputDeclaration {
+import javax.annotation.CheckForNull;
+
+public interface OutputDeclaration extends Statement {
+  Identifier name();
+
+  String type();
+
+  @CheckForNull
+  String condition();
+
+  @CheckForNull
+  String copyCount();
+
+  @CheckForNull
+  String copyInput();
+
+  @CheckForNull
+  String value();
```

Comment:
That's a good question, sometimes we return an `Expression`/`Identifier`, sometimes we return a `String`.
A good example is the `ResourceDeclaration` interface, we also have a mix of `String` and `Identifier`.
We should align, I would be tempted to return the `Expression`/`Identifier` because I'm suspecting that in Bicep we may get something really different, while in `JSON` we always have `String` (even if sometimes they are not "just string", for example: `"concatToVar": "[concat(variables('stringVar'), '-addtovar') ]"`).